lib/jxl/encode.cc

```diff
@@ -191,6 +207,21 @@ JxlEncoderStatus JxlEncoderSetBasicInfo(JxlEncoder* enc,
       return JXL_ENC_ERROR;
       break;
   }
+  if (info->have_animation) {
+    enc->metadata.m.have_animation = true;
+    enc->metadata.m.animation.have_timecodes = false;
```

Comment:
This would better take the value from the user: enc->metadata.m.animation.have_timecodes = info->have_timecodes

Reply:
I have it set to false because the API doesn't currently have a way to set timecodes in the frame header.